OCPBUGS#42519: Clarify Operator group RBAC hash values (#90580)
openshift/openshift-docs: modules/olm-operatorgroups-rbac.adoc

[id="olm-operatorgroups-rbac_{context}"]
= Role-based access control

> When an Operator group is created, three cluster roles are generated. When the cluster roles are generated, they are automatically suffixed with a hash value to ensure that each cluster role is unique. Each Operator group contains a single aggregation rule with a cluster role selector set to match a label, as shown below:

Reviewer: ```suggestion
When an Operator group is created, three cluster roles are generated. When the cluster roles are generated, they are automatically suffixed with a hash value to ensure that each cluster role is unique. Each Operator group contains a single aggregation rule with a cluster role selector set to match a label, as shown in the following table:
```

|`olm.opgroup.permissions/aggregate-to-view: <operatorgroup_name>`
|===

[NOTE]
====

> If you want to use the cluster role of an Operator group to assign role-based access control (RBAC) to a resource, run the following command to get the full name of cluster role and hash value:

Reviewer: For you consideration
```suggestion
To use the cluster role of an Operator group to assign role-based access control (RBAC) to a resource, get the full name of cluster role and hash value by running the following command:
```
